C4067: unexpected tokens following preprocessor directive - expected a newline

Hi, I'm using this template under Windows 11, using Visual Studio Community 2022 as a C++ compiler.

I tried a "dry run" by keeping the original source code and renaming the template to "CustomAudioNodes", following instructions.

During the ```cmake --build CustomAudioStreams-build --parallel``` step I get the following error:
```
C:\Dev\GDExtension\CustomAudioStreams\src\GDExtensionTemplate.cpp(12,68): error C2220: the following warning is treated as an error [C:\D
ev\GDExtension\CustomAudioStreams-build\CustomAudioStreams.vcxproj]
C:\Dev\GDExtension\CustomAudioStreams\src\GDExtensionTemplate.cpp(12,68): warning C4067: unexpected tokens following preprocessor directi
ve - expected a newline [C:\Dev\GDExtension\CustomAudioStreams-build\CustomAudioStreams.vcxproj]
```

This location corresponds to the ```( 2, 1, 0 )``` parameters following the example macro use:
```
#if GDEXTENSIONTEMPLATE_VERSION > GDEXTENSIONTEMPLATE_VERSION_CHECK( 2, 1, 0 )
```

Did I misconfigure something? I can see various ways to work my way around this (removing the example macro call, modifying ```CompilerWarnings.cmake```) but this seemed worth reporting.

--- a/src/GDExtensionTemplate.cpp
+++ b/src/GDExtensionTemplate.cpp
@@ -8,11 +8,6 @@
 /// @file
 /// GDExtensionTemplate example implementation.
 
-// Example of how to use GDEXTENSIONTEMPLATE_VERSION_CHECK() for version checking
-#if GDEXTENSIONTEMPLATE_VERSION > GDEXTENSIONTEMPLATE_VERSION_CHECK( 2, 1, 0 )
-constexpr int DECLARE_SOMETHING = 42;
-#endif
-
 /*!
 @brief Get the version string for this extension.
 

--- a/src/Version.h.in
+++ b/src/Version.h.in
@@ -4,10 +4,6 @@
 
 #include <string_view>
 
-// The version number of this extension. Used for #if comparisons.
-// This is generated using the version set in the CMake project macro.
-#define ${UPPER_PROJECT_NAME}_VERSION  ${UPPER_PROJECT_NAME}_VERSION_CHECK( ${PROJECT_VERSION_MAJOR}, ${PROJECT_VERSION_MINOR}, ${PROJECT_VERSION_PATCH} )
-
 // Creates a version number for use in macro comparisons.
 //
 // Example:
@@ -23,6 +19,10 @@
 #define ${UPPER_PROJECT_NAME}_VERSION_MAJOR  ${PROJECT_VERSION_MAJOR}
 #define ${UPPER_PROJECT_NAME}_VERSION_MINOR  ${PROJECT_VERSION_MINOR}
 #define ${UPPER_PROJECT_NAME}_VERSION_PATCH  ${PROJECT_VERSION_PATCH}
+
+// The version number of this extension. Used for #if comparisons.
+// This is generated using the version set in the CMake project macro.
+#define ${UPPER_PROJECT_NAME}_VERSION  ${UPPER_PROJECT_NAME}_VERSION_CHECK( ${PROJECT_VERSION_MAJOR}, ${PROJECT_VERSION_MINOR}, ${PROJECT_VERSION_PATCH} )
 
 namespace VersionInfo {
     // Project name and version as a string.

